README.md

```diff
@@ -1,38 +1,7 @@
-# Open Voice Network Technical Docs
 
-This repository contains the published and work in progress artifacts, docs, schemas, technical committee working group meeting minutes, etc of the OVN. 
```

Comment:
Can you leave these two lines? Having an overview of what the repo is about is typical of repo README.

Reply:
I added back-in to the PR.  Likely a round about way but they should be back in